pkg/collector/corechecks/containers/kubelet/provider/probe/provider_test.go

```diff
@@ -274,7 +275,8 @@ func TestProvider_Provide(t *testing.T) {
 			fakeTagger := taggerimpl.SetupFakeTagger(t)
 			defer fakeTagger.ResetTagger()
 			for entity, tags := range probeTags {
-				fakeTagger.SetTags(entity, "foo", tags, nil, nil, nil)
+				entityID, _ := taggertypes.NewEntityIDFromString(entity)
```

Comment:
Here also, we may want to change the type of `probeTags` above in this file from `map[string][]string` to `map[EntityID][]string` to be able to get rid of `NewEntityIDFromString` invocations here.